src/relay/pass/quantize/realize.cc

```diff
@@ -97,7 +98,9 @@ inline Expr ForwardOp(const Call& ref_call, const Array<Expr>& args) {
 
 
 /* calculate `data * s1 / s2`, use shift if possible */
-inline Expr MulAndDiv(Expr data, float s1, float s2, DataType dtype) {
+inline Expr MulAndDiv(Expr data, float s1, float s2, DataType dtype,
```

Comment:
This seems like a requantize operation? Is it?

Maybe, we should directly call QNN Requantize operator then. And hide the power of 2 handling inside the requantize/Fixed point multiplication. Special power of 2 handling will help QNN as well. I am fine this PR going in first, and later refactoring. Just putting the point across. Requantize will be a better name for the method.